scripts/create_tables.sql

```diff
@@ -1 +1,27 @@
 -- create doctors, patients, and appointments tables
+CREATE TABLE doctors(
+  id SERIAL PRIMARY KEY,
+  family_name TEXT,
+  given_name TEXT,
+  gender TEXT,
+  born_on DATE,
+  specialty TEXT,
```

Comment:
You have one too many commas here, which SQL will get mad at. Otherwise it is good.